core/src/main/java/org/infinispan/xsite/commands/XSiteStateTransferFinishSendCommand.java

```diff
@@ -0,0 +1,76 @@
+package org.infinispan.xsite.commands;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.util.ByteString;
+import org.infinispan.util.concurrent.CompletableFutures;
+import org.infinispan.xsite.XSiteReplicateCommand;
+import org.infinispan.xsite.statetransfer.XSiteStateTransferManager;
+
+/**
+ * Finish sending XSite state.
+ *
+ * @author Ryan Emerson
+ * @since 11.0
+ */
+public class XSiteStateTransferFinishSendCommand extends XSiteReplicateCommand {
```

Comment:
This command is local site only. It doesn't need to extend XSiteReplicateCommand.

Reply:
I was trying to reduce the amount of duplication, but you're right extending `XSiteReplicateCommand` is not the way todo it.